components/MonthSelection.js

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react'
+
+const MonthSelection = ({onMonthChange, onYearChange}) => {
+    const [month, setMonth] = useState(0) //estado inicial é janeiro
+    const [year, setYear] = useState(2020)
+
+    //skips to the next month, can also handle if it is December
+    const nextMonth = () => {
+        if (month===11) {
+            //if it is December (11), we set the month back to January (0), and we change year.
+            setMonth(0)
+            setYear(year + 1)
+            onYearChange //calls function onYearChange??
```

Comment:
```suggestion
            onMonthChange(0);
            onYearChange(year + 1);
```

Functions passed as arguments still have to be called using parenthesis. This also applies for the previousMonth function.